sdk/evm/src/ntt.ts

```diff
@@ -50,17 +51,48 @@ export class EvmNttWormholeTranceiver<N extends Network, C extends EvmChains>
     );
   }
 
+  getAddress(): ChainAddress<C> {
+    return { chain: this.manager.chain, address: toUniversal(this.manager.chain, this.address) };
+  }
+
   encodeFlags(flags: { skipRelay: boolean }): Uint8Array {
     return new Uint8Array([flags.skipRelay ? 1 : 0]);
   }
 
-  async *setPeer(peer: ChainAddress<C>) {
+  async *setPeer<P extends Chain>(peer: ChainAddress<P>): AsyncGenerator<EvmUnsignedTransaction<N, C>> {
     const tx = await this.transceiver.setWormholePeer.populateTransaction(
       toChainId(peer.chain),
       universalAddress(peer)
     );
     yield this.manager.createUnsignedTx(tx, "WormholeTransceiver.registerPeer");
   }
+
+  async getPeer<C extends Chain>(chain: C): Promise<ChainAddress<C> | null> {
+    const peer = await this.transceiver.getWormholePeer(toChainId(chain));
+    const peerAddress = Buffer.from(peer.substring(2), 'hex');
+    const zeroAddress = Buffer.alloc(32);
+    if (peerAddress.equals(zeroAddress)) {
+      return null;
+    }
```

Comment:
We should prefer Uint8Array over Buffer

```suggestion
    const peer = await this.transceiver.getWormholePeer(toChainId(chain));
    const peerAddress = encoding.hex.decode(peer);
    const zeroAddress = new Uint8Array(32);
    if (encoding.bytes.equals(zeroAddress, peer)) {
      return null;
    }
```

An alternate approach might be to just `BigInt(peer) === 0n` since BigInt will parse hex

```
> Buffer.alloc(32).toString("hex")
'0000000000000000000000000000000000000000000000000000000000000000'
> BigInt(Buffer.alloc(32).toString("hex"))
0n
```

Reply:
also works with 0x prefix
```
> BigInt("0x"+Buffer.alloc(32).toString("hex"))
0n
```